gci/views.py

```diff
@@ -48,6 +49,15 @@ def index(request):
              .format(unix=timegm(datetime.utcnow().utctimetuple()),
                      timestamp=timestamp))
 
+    repo_slug = os.environ.get('TRAVIS_REPO_SLUG')
```

Comment:
could you move the logic to a function so it is re-usable, and we can move the placement around.  separation of data and presentation.

(or maybe even create a django-travis pypi package which makes this information available.

c.f. https://github.com/jayvdb/jekyll-netlify ; we could create a django-netlify also ;-) )